reloj_checador_team03/config/database.yml

```diff
@@ -24,8 +24,8 @@ default: &default
 development:
   <<: *default
   database: reloj_checador_team03_development
-  username: mike
-  password: tscom2020
+  username: fego
+  password: kurosu
   host: localhost
   port: 5432
 
```

Comment:
It is recommended that this file be placed outside of git, in .gitignore

And just leave an example file, to know what is needed to make the connection. With the name of:

`database.yml.example`

Remember the importance of using ENVironment variables.